example/readProperty/readProperty.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"time"
+
+	"github.com/absmach/bacnet/pkg/bacnet"
+	"github.com/absmach/bacnet/pkg/encoding"
+	"github.com/absmach/bacnet/pkg/transport"
+)
+
+func main() {
+	netType := encoding.IPV4
+	destination := bacnet.NewAddress(0, nil, "127.0.0.6:47809", &netType)
```

Comment:
What is this IP address? Define it somewhere.

Reply:
No, this is just an example for testing purposes